docs/business.yaml

```diff
@@ -1564,15 +1564,604 @@ paths:
+                      type: regular
+              amalgamation-short-form-horizontal-request:
+                  summary: Amalgamation Short Form Horizontal Request
+                  value:
+                    filing:
+                        header:
+                          accountId: 3041
+                          certifiedBy: full name
+                          email: "no_one@never.get"
+                          date: '2024-07-26'
+                          name: amalgamationApplication
+                          amalgamationApplication:
+                        amalgamatingBusinesses:
+                          - identifier: BC0883189
+                            role: holding
+                            type: lear
+                          - identifier: BC0883230
+                            role: amalgamating
+                            type: lear
+                        contactPoint: 
+                          email: "contact@example.com"
+                          phone: (555) 555-5555
+                        nameRequest: 
+                            legalType: BC
+                        parties:
+                          - mailingAddress:
+                              addressCity: Victoria
+                              addressCountry: Canada
+                              addressCountryDescription: Canada
+                              addressRegion: BC
+                              deliveryInstructions: ''
+                              postalCode: V8Z 5C5
+                              streetAddress: 3950 Helen Rd
+                              streetAddressAdditional: ''
+                            officer:
+                              email: null
+                              firstName: Completing
+                              lastName: Party
+                              partyType: person
+                            roles:
+                              - appointmentDate: '2024-07-24'
+                                roleType: Completing Party
+                        type: horizontal
+              amalgamation-short-form-vertical-request:
+                summary: Amalgamation Short Form Vertical Request
+                value:
+                  filing:
+                    header:
+                      accountId: 3041
+                      certifiedBy: full name
+                      email: "no_one@never.get"
+                      date: '2024-07-26'
+                      name: amalgamationApplication
+                    amalgamationApplication:
+                      amalgamatingBusinesses:
+                        - identifier: BC0883189
+                          role: primary
+                          type: lear
+                        - identifier: BC0883230
+                          role: amalgamating
+                          type: lear
+                      contactPoint: 
+                        email: "contact@example.com"
+                        phone: (555) 555-5555
```

Comment:
Based on the request body in the Postman collection, missing the `courtApproval` here